spec/services/metalv1/oas3.fetched/components/schemas/AWSFabricProvider.yaml

```diff
@@ -0,0 +1,12 @@
+properties:
```

Comment:
We should probably stop tracking the fetched spec in git.  It makes these PRs more difficult to review.

Reply:
That's fair.